### DISCUSSION TITLE
no ips on docker containers

### DISCUSSION THREAD

**Discussion by WhoDats** [Security-Onion-Solutions/securityonion#8349] [Unsupported Versions]
Hi, I have an air-gapped Security Onion 2.3.130 standalone deployment hosted on ESXI. After restarting SO, vmware-tools stripped away all the docker ips. I deleted the /etc/vmware-tools/scripts/vmware/network script. Any ideas on how to fix the containers now?

**Comment by dougburks** [comment]
Have you tried restarting the containers?

Have you tried rebooting the box?

**Reply by WhoDats** [reply]
Hi Doug,
I have tried restarting the containers and restarting the server, all the containers are stuck on a WAIT START status and cannot be contacted

**Reply by dougburks** [reply]
Have you made any other changes to the box?
